momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java

```diff
@@ -23,7 +22,7 @@ static void beforeAll() {
   }
 
   private static void ensureTestCacheExists() {
-    final CredentialProvider credentialProvider = new EnvVarCredentialProvider("TEST_AUTH_TOKEN");
+    final CredentialProvider credentialProvider = CredentialProvider.fromEnvVar("TEST_AUTH_TOKEN");
```

Comment:
In the other SDKs all of the cred provider constructors provide a way to override the control and data plane endpoints, including these factory functions.  I do think we will need that here but it is a backward compatible addition, so not a blocker for this PR.